README.md

```diff
@@ -48,57 +48,81 @@ We love and welcome contributions to our front-end repository which can be found
 npm install
 ```
 
-## 2. Set up Husky Hooks 🐕️
+<!-- ## 2. Set up Husky Hooks 🐕️
+
+/*This isn't implemented as a script anywhere */
 
 This will automatically run linting before commits and reduce common contributor mistakes.
 
 ```sh
 npm run prepare
-```
+``` -->
 
-## 3. Setting up the Database 🐘
+## 2. Setting up the Database 🐘
 
 For simplicity, we use firebase!
 
 For security, we run a local database so we don't break production!
 
 (optional) You can point to different database environments simply by changing the firebase url and variables in the `.env.dev` file.
 
-### 3.a Setup Firebase CLI
+### 2.a Setup Firebase CLI
 
 #### a.1 Install the Firebase CLI if you do not have it installed on your machine:
 
 ```sh
 npm -g i firebase-tools
 ```
 
-#### a.2 Install Java
+#### a.2 Install Java to the path
 
 on a mac, we recommend using [homebrew](https://brew.sh/)
 
 ```sh
-brew install java
+brew install openjdk
 ```
 
 on a windows pc, we recommend using [chocolatey](https://chocolatey.org/)
 
 ```sh
-choco install java
+choco install openjdk
```

Comment:
If you see the documentation of the firestore local emulator, they mention to install Java JDK. So I'm not sure if it's a good idea to go for `openJDK` . 

So in this part I agree that we should specify the the version of Java that needs to be installed so we need don't need the upgrading part in the next part.

I think it's better to stick with the firestore local emulator documentation: https://firebase.google.com/docs/emulator-suite/install_and_configure

Reply:
So just delete this altogether. Because running choco install java gives this error

```Chocolatey v2.2.2
Installing the following packages:
java
By installing, you accept licenses for the packages.
java not installed. The package was not found with the source(s) listed.
 Source(s): 'https://community.chocolatey.org/api/v2/'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Please see https://docs.chocolatey.org/en-us/troubleshooting for more
 assistance.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - java - java not installed. The package was not found with the source(s) listed.
 Source(s): 'https://community.chocolatey.org/api/v2/'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Please see https://docs.chocolatey.org/en-us/troubleshooting for more
 assistance.


```

Reply:
@Anissa3005 @NabbeunNabi I think that this level of detail is something we should exclude from the README altogether. The steps for installing Firebase CLI will change over time and we don't want to have to keep updating our README to reflect those changes. Instead, we should just tell users to follow the steps in the official documentation and link them to it.

Reply:
@Anissa3005 mentioned that she and Evan put this here because the Firebase CLI doesn't mention the steps for installing Java and kind of assumes you have it. So we thought it might be better to make it less but also keep it for those who are unfamiliar at all with Java. Anissa also said the chocolately command I linked is the one she used and the previous command was incorrect